staging/src/k8s.io/kube-apiextensions-server/test/integration/testserver/resources.go

```diff
@@ -100,20 +118,20 @@ func NewCurletInstance(namespace, name string) *unstructured.Unstructured {
 	}
 }
 
-func CreateNewCustomResourceDefinition(customResourceDefinition *apiextensionsv1alpha1.CustomResourceDefinition, apiExtensionsClient clientset.Interface, clientPool dynamic.ClientPool) (*dynamic.Client, error) {
-	_, err := apiExtensionsClient.Apiextensions().CustomResourceDefinitions().Create(customResourceDefinition)
+func CreateNewCustomResourceDefinition(crd *apiextensionsv1alpha1.CustomResourceDefinition, apiExtensionsClient clientset.Interface, clientPool dynamic.ClientPool) (*dynamic.Client, error) {
+	_, err := apiExtensionsClient.Apiextensions().CustomResourceDefinitions().Create(crd)
 	if err != nil {
 		return nil, err
 	}
 
 	// wait until the resource appears in discovery
-	err = wait.PollImmediate(30*time.Millisecond, 30*time.Second, func() (bool, error) {
-		resourceList, err := apiExtensionsClient.Discovery().ServerResourcesForGroupVersion(customResourceDefinition.Spec.Group + "/" + customResourceDefinition.Spec.Version)
+	err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
```

Comment:
30sec gives you a lot of audit spam with 30msec interval. Wondering, was audit always enabled in integration tests?